met-web/src/components/engagement/view/EngagementBanner.tsx

```diff
@@ -1,20 +1,25 @@
 import React, { useContext } from 'react';
 import { ActionContext } from './ActionContext';
-import BannerWithImage from './BannerWithImage';
-import BannerWithoutImage from './BannerWithoutImage';
-import { Skeleton } from '@mui/material';
+import { Button, Grid, Skeleton } from '@mui/material';
+import { Banner } from '../banner/Banner';
+import { useNavigate } from 'react-router-dom';
 
 export const EngagementBanner = () => {
+    const navigate = useNavigate();
     const { engagementLoading, savedEngagement } = useContext(ActionContext);
 
-    const imageExists = !!savedEngagement.banner_url;
-
+    const surveyId = savedEngagement.surveys[0]?.id || '';
```

Comment:
I don't love that we are prepping for multiple surveys but there's stuff that is still fixed for only one, its just a comment no change for now

Reply:
I understand, but it wasn't really a decision here. I noticed that this  db.relationship('Survey', backref='engagement', cascade='all, delete') returns an array. So I continued handling it like that and thought to keep it open for multiple surveys.